bgpd/bgp_debug.c

```diff
@@ -2373,6 +2406,11 @@ static int bgp_config_write_debug(struct vty *vty)
 		write++;
 	}
 
+	if (CONF_BGP_DEBUG(cond_adv, COND_ADV)) {
+		vty_out(vty, "debug bgp cond-adv\n");
```

Comment:
Can we have a _normal_ naming here? It's easier to read without guessing what it means.

Reply:
you mean 'conditional-advertisement' instead of 'cond_adv' ?

Reply:
yep

Reply:
sure